### PR TITLE
Dashboard: Implement active parkings table

### PR DESCRIPTION
**Pull request by suutari-ai**
Add monitoring API endpoint for listing valid parkings at given time and
use that to list the active parkings at selected time and region in the
Dashboard.

Refs HP-35

### CODE REVIEW

File: parkings/api/monitoring/serializers.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    operator_name = serializers.CharField(
+        source='operator.name', read_only=True)
+
+    class Meta:
+        model = Parking
+        fields = [
+            'id',
+            'registration_number',
+            'location',
+            # 'parking_area',
```
Comment (tuomas777) [inline parkings/api/monitoring/serializers.py]: Is this intentionally left here? And if it is, I don't really see the point.

File: parkings/api/monitoring/valid_parking.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        ]
+        strict = django_filters.STRICTNESS.RAISE_VALIDATION_ERROR
+
+    def filter_time(self, queryset, name, value):
+        return queryset.valid_at(value)
+
+
+class ValidParkingViewSet(viewsets.ReadOnlyModelViewSet):
+    permission_classes = [MonitoringApiPermission]
+    queryset = Parking.objects.all().order_by('time_start')
```
Comment (tuomas777) [inline parkings/api/monitoring/valid_parking.py]: This could have `select_related('operator')` as the serializer is using `operator.name` ?

Comment (suutari-ai) [reply parkings/api/monitoring/valid_parking.py]: Good point. Will add.

File: parkings/api/monitoring/serializers.py (hunk 1 above)
Comment (suutari-ai) [reply parkings/api/monitoring/serializers.py]: Yeah, there's no much use having it there as a comment. I will remove it.

**Pull request merged**